ca_description/urdf/dock/dock.xacro

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" ?>
+<robot name="dock" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find ca_description)/urdf/common_properties.xacro" />
+  <xacro:macro name="dock" params="width">
+    <link name='back'>
+      <visual name='visual'>
+        <origin xyz="0 0 0" rpy="0 0 0" />
```

Comment:
Origin tags with all their components in `0` are not necessary since its the default value, so please remove them.